### PR TITLE
DNS Records - documentation edited (note multiple ???'d questions)

### PR DESCRIPTION
**Pull request by alexander-levitski**

### CODE REVIEW

File: community/cloud-foundation/templates/dns_records/README.md [hunk 1]
```diff
@@ -16,50 +16,49 @@ This Template manages Cloud DNS records via record-sets
...
-- [DNS Records](dns_records.py.schema)
+- [DNS records](dns_records.py.schema)
 
 ### Usage
 
-1. Clone the [DM Samples_repository](https://github.com/GoogleCloudPlatform/deploymentmanager-samples):
+1. Clone the [Deployment Manager samples repository](https://github.com/GoogleCloudPlatform/deploymentmanager-samples):
 
-```bash
+```
```
Comment (sourced-vince) [inline community/cloud-foundation/templates/dns_records/README.md]: shell?

Comment (alexander-levitski) [reply community/cloud-foundation/templates/dns_records/README.md]: Yep, shell

File: community/cloud-foundation/templates/dns_records/dns_records.py [hunk 2]
```diff
@@ -11,22 +11,22 @@
...
 
 
 def generate_config(context):
     """ Entry point for the deployment resources.
-    For each ResourceRecordSet. Create:
-    1. A Change to create it.
-    2. A Change to delete it.
+    For each ResourceRecordSet, create:
+    1. A change to create it.
+    2. A change to delete it. ??? Not sure what these 1 and 2 mean. Please rephrase. ???
```
Comment (sourced-praveenc) [inline community/cloud-foundation/templates/dns_records/dns_records.py]: Not sure how else could I phrase this? - This not clear?
`For each ResourceRecordSet, create: ...`

Also, if/when you make changes to comments on the `python` files - please make sure to run `yapf` or `pylint` before committing to ensure there are no linting errors.

File: community/cloud-foundation/templates/dns_records/dns_records.py.schema [hunk 3]
```diff
@@ -77,11 +77,11 @@ properties:
...
         ttl:
           type: integer
-          description: The time-to-live cache in seconds.
+          description: The time-to-live cache, in seconds.
           minimum: 0
         rrdatas:
           type: array
           description: |
-            As defined in RFC 1035 (section 5) and RFC 1034 (section 3.6.1).
+            A list of ??? what ???, as defined in RFC 1035 (section 5) and RFC 1034 (section 3.6.1).
```
Comment (sourced-praveenc) [inline community/cloud-foundation/templates/dns_records/dns_records.py.schema]: `A list of resourceRecordSets as defined in RFC` ...

File: community/cloud-foundation/templates/dns_records/examples/dns_records.yaml [hunk 4]
```diff
@@ -1,8 +1,8 @@
-# Example on how to use the DNS records template
+# Example of the DNS records template usage.
 #
-# This example creates DNS RecordSets to manage a sets of DNS records.
-# A, AAAA, MX, TXT records are created for a given managed zone 
-# defined via `zoneName` property and DNS name via `dnsName` property.
+# This example creates DNS RecordSets to manage a set of DNS records.
+# A, AAAA, MX, and TXT records are created for a given managed zone, 
+# defined by the `zoneName` property, and a DNS name defined by the `dnsName` property.
```
Comment (sourced-praveenc) [inline community/cloud-foundation/templates/dns_records/examples/dns_records.yaml]: I think we should either remove all the lines starting `A, AAAA ...` as there are more than the those 4 or list all of them that are actually `A, AAAA, MX, TXT, PTR, SPF, SRV` - I'll leave this to you to decide.

File: community/cloud-foundation/templates/dns_records/dns_records.py.schema [hunk 5]
```diff
@@ -35,18 +35,18 @@ properties:
...
     description: |
-      The DNS name of this managed zone, for instance "example.com.".
-      Note: A fully qualified domain name (FQDN) always end with a dot.
+      The DNS name of the managed zone; for example, "example.com."
+      A fully qualified domain name (FQDN) must end with a period "."
   resourceRecordSets:
     type: array
     description: |
-      The ResourceRecordSets meant to be in the zone. See
+      A list pd ResourceRecordSets meant to be in the zone. See
```
Comment (sourced-praveenc) [inline community/cloud-foundation/templates/dns_records/dns_records.py.schema]: What's `A list pd` ?

File: community/cloud-foundation/templates/dns_records/dns_records.py [hunk 6]
```diff
@@ -11,22 +11,22 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Create DNS record-sets resources for a managed zone"""
+""" This template creates DNS records using recordsets. """
```
Comment (sourced-praveenc) [inline community/cloud-foundation/templates/dns_records/dns_records.py]: I doubt it's `using recordsets` ...

File: community/cloud-foundation/templates/dns_records/dns_records.py (hunk 2 above)
Comment (alexander-levitski) [reply community/cloud-foundation/templates/dns_records/dns_records.py]: Sorry if I did not make myself clear. What does this mean: "... create  1. A change to create it; 2. A change to delete it."?

Comment (sourced-praveenc) [reply community/cloud-foundation/templates/dns_records/dns_records.py]: aah .. it's a `action` ... gcp has actions and types .. so
>for each ResourceRecordSet, create a resource with an action to `create` and another resource with an action to `delete`

Hope that clarifies.

**Pull request closed without merging**